### DISCUSSION TITLE
Configure the inverter voltage/frequency/current for each phase

### DISCUSSION THREAD

**Discussion by einsminion** [slipx06/sunsynk-power-flow-card#202] [General]
Hi,
can be added the inverter voltage/frequency/current for each phase in case of tree phase inverters?

inverter_voltage_154
load_frequency_192
inverter_current_164

Regards!

**Comment by slipx06** [comment]
Hi, You can do this for current and voltage. Have a look through the documentation https://slipx06.github.io/sunsynk-power-flow-card/configuration.html

The entities you need:
```

inverter_voltage_154:
inverter_voltage_L2: 
inverter_voltage_L3: 
inverter_current_164:
inverter_current_L2: 
inverter_current_L3: 

```
